### ISSUE TITLE
minilm_jit example doesn't work

### ISSUE THREAD

**Issue by powderluv** [NodLabs/SHARK#32]
```
(shark.venv) a@debian-1:~/github/dshark$ python -m  shark.examples.minilm_jit
/home/a/github/dshark/shark.venv/lib/python3.7/site-packages/torch/nn/modules/module.py:1403: UserWarning: positional arguments and argument "destination" are deprecated. nn.Module.state_dict will not accept them in the future. Refer to https://pytorch.org/docs/master/generated/torch.nn.Module.html#torch.nn.Module.state_dict for details.
  " and ".join(warn_msg) + " are deprecated. nn.Module.state_dict will not accept them in the future. "
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at microsoft/MiniLM-L12-H384-uncased and are newly initialized: ['classifier.weight', 'classifier.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
Target triple found:x86_64-linux-gnu
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
        - Avoid using `tokenizers` before the fork if possible
        - Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
(shark.venv) a@debian-1:~/github/dshark$    
```

**Comment by powderluv** [comment]
Does no error on the output mean successful run ?

**Comment by powderluv** [comment]
maybe we can run the inference a few times and output the time it took to run similar to:

https://github.com/powderluv/transformer-benchmarks/blob/f8258f751cd3b7e87bf66ffc1b38e4443a70c1e3/benchmark.py#L318-L339

**Comment by pashu123** [comment]
Sounds good.

**Comment by pashu123** [comment]
Added here: https://github.com/NodLabs/dSHARK/pull/33/files

**Comment by powderluv** [comment]
Can we also please add a number of iteration flag ?

**Comment by pashu123** [comment]
Iters flag to run https://github.com/NodLabs/dSHARK/blob/db5be15310dda3d8906206d68cc9eb7136373bac/shark/examples/minilm_jit.py#L30

**Comment by pashu123** [comment]
Basically no. of times to run the inference?

**Comment by powderluv** [comment]
yeah

**Comment by powderluv** [comment]
maybe even a number of warmup runs
